Data_Analysis_Markdown.Rmd

```diff
@@ -0,0 +1,503 @@
+                             "Belize"))
+
+#figure 2
+p <- ggplot(data = mean_ed_graph, aes(x = colorr_recode, y = mean_ed, color = pais))
+fig2 <- p + layer (geom="line", size = 1) +
+  facet_wrap(~region, scales = "free") +
+  scale_color_manual("Countries, by Region\n",
+                     labels = c("CENTRAL AMERICA & MEXICO\nPanama", 
+                                  "Costa Rica", 
+                                  "Honduras",  
+                                  "Mexico", 
+                                  "Guatemala", 
+                                  "El Salvador", 
+                                  "Nicaragua", 
+                                "ANDEAN\nBolivia", 
+                                  "Peru", 
+                                  "Venezuela", 
+                                  "Colombia", 
+                                  "Ecuador",
+                                "SOUTHERN CONE & BRAZIL\nArgentina", 
+                                  "Chile", 
+                                  "Paraguay", 
+                                  "Uruguay",  
+                                  "Brazil", 
+                                "CARIBBEAN\nSuriname", 
+                                  "Jamaica", 
+                                  "Trinidad & Tobago",  
+                                  "Guyana", 
+                                  "Dominican Republic", 
+                                  "Belize"),
+                     values = c("dark red", 
+                                "orange", 
+                                "gold", 
+                                "dark blue", 
+                                "sky blue", 
+                                "dark green", 
+                                "light green",
+                                "dark red", 
+                                "sky blue", 
+                                "dark green", 
+                                "orange", 
+                                "gold",
+                                "dark red", 
+                                "sky blue", 
+                                "dark green", 
+                                "orange", 
+                                "gold", 
+                                "orange", 
+                                "dark blue", 
+                                "gold", 
+                                "dark green", 
+                                "dark red", 
+                                "sky blue")) +
+  xlab("Skin Shade") + 
+  ylab("Mean Years of Education") +
+  scale_y_continuous(limits = c(5,13), breaks = (5:13)) +
+  scale_x_continuous(limits = c(1,11), breaks = (1:11), 
+      labels = c("1-lightest", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11-darkest")) +
+  ggtitle("Relation between Skin Color and Educational Attainment 
+          \n\nin Latin America and the Caribbean") +
+  theme(plot.title = element_text(lineheight =.6, face = "bold", size = 16),
```

Comment:
This is a really nice graph and good use of themes in ggplot2! Commenting on what some of these them options are doing, might be useful for looking at this years from now or when you want to apply it to a different graph on another paper.